### DISCUSSION TITLE
Installing plugin while only deploying threw GitHub

### DISCUSSION THREAD

**Discussion by Staubgeborener** [mkdocs/mkdocs#3591] [Q&A] [answered]
I am using mkdocs as a seperate branch in a repository which is working fine. I can change my code anytime and it will be automatically deployed with a GitHub workflow. Now i want to use a plugin like [mkdocs-video](https://pypi.org/project/mkdocs-video/) which needs to be install with `pip install mkdocs-video`. Any chance to install it without running mkdocs locally on a system while all datas are only stored inside a GitHub repository?

**Comment by pawamoy** [comment, marked as answer]
Yes, install it in your GitHub workflow? Not sure to understand.
